### ISSUE TITLE
New icon proposal + disabling FFZ

### ISSUE THREAD

**Issue by Zabriskije** [strumswell/BTTV-for-Safari#2]
Hi @strumswell!
I've made a new app icon to thank you for taking BTTV and FFZ to Safari. In case you like it, feel free to use it.
I've also included in [this zip](https://github.com/strumswell/BTTV-for-Safari/files/10166679/BTTV.zip) the logo alone for the toolbar.

<img width="128" src="https://user-images.githubusercontent.com/101254295/205918073-afa6dffd-40b7-4f81-8cfe-edcc9338392a.png">

And, to not open another issue: in case I don't want to use FFZ, there's a way I can disable it?
I've tried removing the injection from `content.js` located in `/BTTV for Safari.app/Contents/PlugIns/BTTV for Safari Extension.appex/Contents/Resources`:

`// Inject FFZ script`
`const ffzScript = document.createElement('script');`
`ffzScript.type = 'text/javascript';`
`ffzScript.src = 'https://cdn.frankerfacez.com/static/ffz_injector.user.js';`
+
`head.appendChild(ffzScript);`

but if I do that, Safari reports that the entry "icons" in manifest.js failed to load (wut?).

**Comment by strumswell** [comment]
Thank you SO much for spending time on creating such an amazing app icon. I will definitely use that one in an updated version, including your request concerning some toggles for the FFZ and BTTV script injection.

Please give me some time. I will try to incorporate these changes this weekend and will probably also upload it to the App Store as a second distribution channel. (I still have a paid dev account lying around)

Let me say it again: THANK YOU. 🎉

**Comment by Zabriskije** [comment]
Whoa! 😳 Glad you liked it.
Unfortunately I've just realized that is a bit too flat, so I'm sorry but here's a better one:

<img width="128" src="https://user-images.githubusercontent.com/101254295/206060432-2608fc58-bab9-4a96-9cd1-7219990b9052.png">

Take your time, download the [new zip](https://github.com/strumswell/BTTV-for-Safari/files/10171804/BTTV.zip), and hit me up if you need something else.

**Comment by strumswell** [comment]
Thank you again for the logo and your feature request. I included the new app/ toolbar icon and added your request regarding disabling FFZ. If you click on the toolbar icon, a popup should appear. In there, you can simply check or uncheck BTTV or FFZ to get them injected or not when visiting Twitch.

I still have a few minor things to iron out but I want you to test this release if you are free. :) Let me know what you think and if you have any issues.

Download: [BTTV for Safari.zip](https://github.com/strumswell/BTTV-for-Safari/files/10180767/BTTV.for.Safari.zip)

**Comment by Zabriskije** [comment]
Tried out, everything is working properly, good job!
The toolbar icon seems too thick to me, what do you think?
I've made a thinner one in case you agree:

<img width="64" src="https://user-images.githubusercontent.com/101254295/206687279-5be25c6b-63f0-4d43-9cac-95ac4aa86c35.png">

**Comment by strumswell** [comment]
Yeah, I noticed this as well but could live with it. Good that you thought so as well, because the thinner variant will definitely look better. ;)

I'll keep you updated.
